pypeit/slittrace.py

```diff
@@ -479,7 +479,7 @@ def get_slitlengths(self, initial=False, median=False):
         slitlen = right - left
         return np.median(slitlen, axis=1) if median else slitlen
 
-    def get_radec_image(self, wcs, alignSplines, tilts, slit_compute=None, slice_offset=None, initial=True, flexure=None):
+    def get_radec_image(self, wcs, alignSplines, tilts, slit_compute=None, slice_offset=None, initial=False, flexure=None):
```

Comment:
Have you followed how this change in the default affects other calls to `get_radec_image`?